packages/logger/test/end-to-end/helpers/split-and-parse-json-logs.js

```diff
@@ -7,11 +7,16 @@
  *     Returns the parsed JSON logs as an array of log objects.
  */
 function splitAndParseJsonLogs(logString) {
-	return logString
-		.trim()
-		.split(/\n+/g)
-		.filter((log) => log.trim())
-		.map((log) => JSON.parse(log));
+	return (
+		logString
+			.trim()
+			.split(/\n+/g)
+			.filter((log) => log.trim())
+			// Node.js sometimes logs deprecation warnings which are not valid JSON.
+			// We need to filter these out so we don't error in our JSON.parse map
+			.filter((log) => !log.toLowerCase().includes('deprecation'))
```

Comment:
Nice addition here!

Reply:
Yep Node.js 22 deprecates some stuff so the tests were failing 😁